File: [프로그래머스] 월간 코드 챌린지 시즌1_이진 변환 반복하기.py
def solution(s):
    binCnt = 0
    cnt = 0
    answer = []
    while s != "1":
        cnt += s.count("0")
        s = s.replace("0", "")
        s = bin(len(s))[2:] 	#앞 부분 '0b' 제거
        binCnt += 1
    answer = [binCnt, cnt]
    return answer

File: [프로그래머스] 월간 코드 챌린지 시즌1_이진 변환 반복하기.js
function solution(s) {
    var answer = [0, 0];
    while(s !== "1"){
        let sArr = s.split("")
        let len = sArr.filter((it)=> it === '1').length;
        answer[1] += sArr.length - len;
        s = len.toString(2);
        answer[0]++;
    }
    return answer;
}